blog/_posts/2020-07-31-prefab-tools.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: "Prefabricated Web Tools"
+date: 2020-06-03
```

Comment:
To-Do: change this date to 2020-07-31. 
Kept at June date for now, so it will show up when testing locally with jekyll.